concepts/Common/.NET Backend Solutions/Web API Service (CRUD)/Web API Service (CRUD).md

```diff
@@ -0,0 +1,49 @@
+---
+title: Backend Web/REST API for JS Apps with Access Control Rights (Powered by EF Core) 
+---
+
+Besides DevExtreme UI components, your client-side JavaScript (JS) applications often require an easy-to-setup and secure Web API backend for CRUD and custom operations with a database, authentication and authorization (role-based access control, permission management). To implement such functionality, you can use [DevExpress .NET App Security & Web API Service](https://www.devexpress.com/products/net/application_framework/security-web-api-service.xml). XAF's Solution Wizard includes a "1-Click" solution to create [ASP.NET Core Web/HTTP API services](https://docs.devexpress.com/eXpressAppFramework/403394/backend-web-api-service) (via OData and Swagger/OpenAPI).
+
+[Read Tutorial](https://docs.devexpress.com/eXpressAppFramework/404449/backend-web-api-service/get-started-with-web-api-service) | [Watch Videos](https://www.youtube.com/watch?v=T7y4gwc1n4w&list=PL8h4jt35t1wiM1IOux04-8DiofuMEB33G)
+
+![Backend Web/REST API](/images/Common/backend-web-api-article-main.png)
+
+#### Benefits
+
+- **Powered by Entity Framework Core ORM (EF Core)**    
+DevExpress Web API Service leverages the functionality of EF Core to implement CRUD operations required for your app.
+
+- **Multiple Authentication Strategies**     
+You application’s authentication mechanism can leverage built-in support for OAuth2, JWT, and OpenID. You can also implement a custom authentication strategy.
+
+- **Cross-Platform Compatibility**     
```

Comment:
Sounds for me like an obvious thing, not like a product feature - any HTTP/Web API can be consumed by different types of clients by design. No reason to highlight this as a feature.